style_guide.md

```diff
@@ -0,0 +1,270 @@
+# Yield Style Guide
+
+This guide is intended to provide coding conventions for writing solidity code. This guide should be thought of as an evolving document that will change over time as useful conventions are found and old conventions are rendered obsolete.
+
+## Code Layout
+1. Pragma statement
+2. Import statements
+3. Interfaces
+4. Libraries
+5. Contract
+
+#### Code Order
+Inside each contract, library or interface, use the following order:
+
+1. Library declarations (using statements)
+2. Constant variables
+3. State variables
+4. Type declarations
+5. Events
```

Comment:
no mention of errors.  we should discuss custom errors policy as a team

Reply:
oh good point. I've actually completely ignored their existence in lieu of require statements.